.github/workflows/functions.yaml

```diff
@@ -0,0 +1,32 @@
+name: Functions deploy test
+
+on: workflow_dispatch
```

Comment:
I had to look up what this meant. Consider adding a link out to https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow or similar